### PR TITLE
#241 - added generated metadata files and PrimaryOutput test

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #241 I've added generated by createrepo metadata files for test rpm packages and test for `PrimaryOutput`.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#272](https://github.com/artipie/rpm-adapter/issues/272) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/e22fe1e2-1964-4ca8-9e28-44f8afa79f08); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/rpm/pkg/PrimaryOutputTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+     * @return The header
+     * @param file Rpm file path
+     * @throws InvalidPackageException In case package is invalid.
+     * @throws IOException On error
+     * @todo #241:30min Introduce class from this method: the method is copied from FilePackage
+     *  class, we need to extract class from it, test it and use it at least in FilePackage,
+     *  ModifiableMetadataTest and here.
+     */
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
+    private Header header(final Path file) throws InvalidPackageException, IOException {
```
Comment (Vatavuk) [inline src/test/java/com/artipie/rpm/pkg/PrimaryOutputTest.java:98]: @olenagerasimova `InvalidPackageException` doesn't need to be declared explicitly. Also I don't think that is a good idea to throw both runtime and catch exceptions. I would throw only `InvalidPackageException` or only `IOException`. wdyt?

**Comment by olenagerasimova** [comment]
@Vatavuk thanks, corrected

**Comment by 0crat** [comment]
Job was finished in 23 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CR, thanks!

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**